draft-ietf-mls-protocol.md

```diff
@@ -1444,8 +1445,10 @@ struct {
             uint32 leaf_index;
         case external:
             uint32 sender_index;
-        case new_member:
+        case new_commit_member:
             struct{};
+        case new_proposal_member:
+            KeyPackageID proposed_member;
```

Comment:
This can just be `struct{}`, since because the KeyPackage you apply is the one in the Add proposal.